wg-deployment/charter.md

```diff
@@ -0,0 +1,51 @@
+# WG Deployment Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+Manage manifests for deploying and operating Kubeflow.
+Develop and maintain tools like kfctl (CLI + operator) for using the manifests to deploy Kubeflow.
+
+### In scope
+
+#### Responsibilities
+
+##### Application Owners
```

Comment:
Under responsibilities:

How about maintaining a test suite to ensure all application manifests satisfy basic requirements? e.g. ensuring they aren't using obsolete features in kustomize, Kubernetes, ISTIO etc... (we already have some minimal tests)?